WORKSPACE

```diff
@@ -133,7 +133,7 @@ container_pull(
     digest = "sha256:e357a240b1e8668c851a1dcb3e3244b4e037aab56029f394dda6925070d473a9",
     registry = "gcr.io",
     repository = "pipecd/piped-base",
-    tag = "0.1.0",
+    tag = "0.1.1",
```

Comment:
This should be updated in another PR after this image was pushed.
And the digest must be updated too.

Reply:
Ah, I got wrong! Thanks.

Reply:
Is it no problem to keep `dockers/piped-base/DOCKER_BUILD` as is?

Reply:
Yes, that file is for defining the version of that docker image. And so that is ok.

Reply:
👍